blockly-core/core/ui/block_space/block_space_editor.js

```diff
@@ -417,23 +418,34 @@ Blockly.BlockSpaceEditor.prototype.svgSize = function() {
  */
 Blockly.BlockSpaceEditor.prototype.svgResize = function() {
   var svg = this.svg_;
-  var style = window.getComputedStyle(svg);
-  var borderWidth = 0;
-  if (style) {
-    borderWidth = parseInt(style.borderLeftWidth, 10) +
-      parseInt(style.borderRightWidth, 10);
-  }
-  var div = svg.parentNode;
-  var width = div.offsetWidth - borderWidth;
-  var height = div.offsetHeight;
-  if (svg.cachedWidth_ != width) {
-    svg.setAttribute('width', width + 'px');
-    svg.cachedWidth_ = width;
-  }
-  if (svg.cachedHeight_ != height) {
-    svg.setAttribute('height', height + 'px');
-    svg.cachedHeight_ = height;
+  var svgStyle = window.getComputedStyle(svg);
+  var svgBorderWidth = 0;
+  if (svgStyle) {
+    svgBorderWidth = parseInt(svgStyle.borderLeftWidth, 10) +
+      parseInt(svgStyle.borderRightWidth, 10);
   }
+
+  // Subtract any pixels present above the svg element from the available height
+  // (only need to do this for mainBlockSpaceEditor's svg element, but fall back
+  // to this.svg_ during mainBlockSpaceEditor's creation)
+  var containerDiv = svg.parentNode;
+  var topmostSvgElement = Blockly.mainBlockSpaceEditor ? Blockly.mainBlockSpaceEditor.svg_ : svg;
```

Comment:
Yeah, I just checked and getSize() for the container is returning the size including the container's border. In order to simulate clientWidth, you'll need to compute containerBorderWidth (like was already taking place for svgBorderWidth) and subtract that out.

But we may be ok just using clientWidth. I checked IE 11 just now and it is fine. Perhaps the only problem was calling offsetTop on the SVG element?